homeassistant/components/motion_blinds/strings.json

```
@@ -28,7 +28,8 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
-      "connection_error": "[%key:common::config_flow::error::cannot_connect%]"
+      "connection_error": "[%key:common::config_flow::error::cannot_connect%]",
+      "dhcp_no_motionblinds": "DHCP discovered device is not a Motion gateway"
```

```suggestion
      "not_motionblinds": "Discovered device is not a Motion gateway"
```

Lets make this more generic in case another discovery method becomes available in the future